init.lua

```diff
@@ -59,19 +59,34 @@ end
 
 -- Usage:
 --   resizer = SkyRocket:new({
+--     opacity = 0.3,
 --     moveModifiers = {'cmd', 'shift'},
+--     moveMouseButton = 'left',
 --     resizeModifiers = {'ctrl', 'shift'}
+--     resizeMouseButton = 'left',
 --   })
 --
+local function buttonNameToEventType(name)
+    if name == 'left' then
+        return hs.eventtap.event.types.leftMouseDown
+    end
+    if name == 'right' then
+        return hs.eventtap.event.types.rightMouseDown
+    end
+    error('only "left" and "right" mouse button supported, got ' .. name)
```

Comment:
**chore**: Please update this error message to include the name of the parameter that they misconfigured (`resizeMouseButton`)